src/libprojplugin/libprojplugin.cpp

```diff
@@ -195,20 +158,22 @@ void LibprojPlugin::triggerAddNewFileAction()
    }
 }
 
-void LibprojPlugin::saveProjectData(const string & WhatToAppend, const string & WhereToAppend)
+
+/* this function isn't needed for now */
```

Comment:
#bug Don't disable code via commit
if you want to save it any where, create snippet, or named stash